src/server.py

```diff
@@ -11,13 +12,15 @@
 from sqlalchemy import and_, func, desc, case
 from database.models import Seta, Testjobs, Dailyjobs
 
-import MySQLdb
 from flask import Flask, request, json, Response, abort
 
 SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
 static_path = os.path.join(os.path.dirname(SCRIPT_DIR), "static")
 app = Flask(__name__, static_url_path="", static_folder=static_path)
 JOBSDATA = jobtypes.Treecodes()
+PORT = int(os.environ.get("PORT", 8157))
+urlparse.uses_netloc.append("postgres")
+DBURL = urlparse.urlparse(os.environ["DATABASE_URL"])
```

Comment:
can we document this better- with a comment and mention in a setup script or a readme